cuFOLIO_portfolio_optimization/CVaR/01_optimization_with_cufolio.ipynb

```diff
@@ -40,10 +40,10 @@
     "# If dependencies are already installed, you can comment out or skip this cell.\n",
     "\n",
     "# Install cuOpt (if not already installed)\n",
-    "#!pip install --upgrade --user --extra-index-url https://pypi.nvidia.com -q cuopt-cu12\n",
+    "# !pip install --upgrade --user --extra-index-url https://pypi.nvidia.com -q cuopt-cu12==25.10.* \n",
     "\n",
     "# Install other dependencies (if not already installed)\n",
-    "#!pip install --user --extra-index-url https://pypi.nvidia.com -q \"numpy>=1.24.4\" \"pandas>=2.2.1\" \"cvxpy>=1.6.5\" \"scipy==1.15.2\" \"scikit-learn==1.6.1\" \"msgpack>=1.1.0\" \"cuml-cu12==25.4.*\" \"seaborn>=0.13.2\" bin/cufolio-25.8-py3-none-any.whl"
+    "!pip install --user --pre --extra-index-url https://pypi.nvidia.com -q \"numpy>=1.24.4\" \"pandas>=2.2.1\" \"cvxpy>=1.6.5\" \"scipy==1.15.2\" \"scikit-learn==1.6.1\" \"msgpack>=1.1.0\" \"cuml-cu12\" \"seaborn>=0.13.2\" bin/cufolio-25.8-py3-none-any.whl"
```

Comment:
do we really want to remove the # here? that's fine, just making sure it wasn't a mistake